### ISSUE TITLE
Bot not working

### ISSUE THREAD

**Issue by ReenigneArcher** [RedditBot-Discord/redditbot#34]
I deleted and recreated my webhooks 3 days ago, but the bot is not messaging the channel when new posts are added to the subreddit.

![image](https://user-images.githubusercontent.com/42013603/120949406-73bc6f80-c712-11eb-84ff-1be036f50459.png)

These were both posted after the webhook was re-created.
![image](https://user-images.githubusercontent.com/42013603/120949513-b3835700-c712-11eb-90c0-7f215896653c.png)

**Comment by mrshmllow** [comment]
this kind of thing isn't an exact science, it's based off what the reddit API gives the bot and when..

However I'll look into the logs and see, You _may_ have to just be patient with it 🤷‍♀️

**Comment by ReenigneArcher** [comment]
Hmm, okay. The bot worked pretty well when the subreddit was new. It's been kind of an inactive subreddit over the last few months and the bot hasn't worked really in maybe two months.

**Comment by mrshmllow** [comment]
yeah, it's been pretty bad for a while now but I have fixed a lot of bugs and it should be working a lot better now

**Comment by ReenigneArcher** [comment]
I replaced this bot with Redditor. I prefer your bot due to the slash commands, but I rely on the bot to alert me when someone on my subreddit is asking for support. Redditor is not instant like this bot used to be, but I get notifed within 30 minutes at the most. Hope you can get the issues sorted and then I may revisit your bot. Thanks!
